--- vendor/larapack/hooks/src/Composer.php	2018-02-02 21:17:22.000000000 +0800
+++ stonecutter/hooks/src/Composer.php	2019-01-19 23:02:16.089861298 +0800
@@ -30,9 +30,11 @@
             $this->items['repositories'] = [];
         }
 
-        $this->items->set('repositories.'.$name, $info);
+        if ($this->items->get('repositories.'.$name) != $info) {
+            $this->items->set('repositories.'.$name, $info);
 
-        $this->changed = true;
+            $this->changed = true;
+        }
 
         return $this;
     }
@@ -72,7 +74,7 @@
     public function save()
     {
         if ($this->changed) {
-            $this->filesystem->put($this->location, $this->encode($this->items->all()));
+            $this->filesystem->put($this->location, $this->encode($this->items->all())."\n");
 
             $this->changed = false;
         }
